src/tree/Element.mjs

```diff
@@ -1596,6 +1596,19 @@ export default class Element {
         }
     }
 
+    /**
+     * isfocused property of the Elemenet
+     *
+     * This property sets and gets if the element has focus
+     */
+    get isfocused() {
```

Comment:
This may shadow the existing hasFocus() and hasFinalFocus on _Component_; is there a way to lean on that existing feature (can a non-Component have focus?? sorry I'm new here :) ).

Reply:
MR with proposed refactor for that raised with jhderojasUVa directly.

Reply:
I second this, there shouldn't be any need to pollute the core with additional flags like `isfocused`.